templates/hawk_apiserver.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
```

Comment:
I am ok to have a separated systemd unit for that. We might need to rethink the name in the future, but it needs to depend on hawk.service, since it is reusing the certificates from that. If don't, it will not work if hawk service was never executed.

Reply:
Totally agree!

Reply:
I think that is OK to be provided by this project on this initial moment, but in the future we should move it to the `hawk-apiserver` project and RPM package.